Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -716,17 +722,18 @@ - (BOOL)isCapturingStillImage {
 
 #pragma mark - Setters
 
-- (void)setCamera:(MTBCamera)camera {
```

Comment:
I'm not sure if breaking people's code at this point is a good idea, or are you planning to go for version 4.0 soon anyway? I would propose the following:

* add `setCamera:error:` to the public interface
* leave the property (and revert removing the setter), but add a comment that setting it may fail silently, and reference the new method – we might even add a deprecation warning?
* change the `README` as suggested